core/load-git.c

```diff
@@ -988,7 +996,7 @@ static void trip_parser(char *line, struct membuffer *str, void *_trip)
 static struct keyword_action settings_action[] = {
 #undef D
 #define D(x) { #x, parse_settings_ ## x }
-	D(autogroup), D(divecomputerid), D(subsurface), D(units), D(userid), D(version),
+	D(autogroup), D(divecomputerid), D(subsurface), D(prefs), D(units), D(userid), D(version)
 };
```

Comment:
the hunk starts just one line too low... the line above this says
`/* These need to be sorted! */`
thankfully I remembered that for some strange reason...
so prefs need to come before subsurface :-)